src/components/tabs/__internal__/tab-title/tab-title.style.ts

```diff
@@ -250,7 +270,16 @@ const StyledTitleContent = styled.span`
   `}
 `;
 
-const StyledTabTitle = styled.button`
+interface StyledTabTitleProps extends Omit<TabTitleProps, "tabIndex"> {
+  tabIndex?: number;
```

Comment:
I'll have a look - I'm pretty sure I had TS errors until I listed them but it's possible that other changes I made since make this OK.

Reply:
I've looked at this and indeed most of these props don't need to be listed. The exception is `tabIndex` which is a `string` in `TabTitleProps` but needs to be a `number` on the element itself. While I could leave that in I've decided to change the props to `TabTitle` since its only an internal component anyway, which means I can just use `TabTitleProps` here and don't need a separate `StyledTabTitleProps` - thanks.